src/ADAL.PCL/Flows/AcquireTokenInteractiveHandler.cs

```diff
@@ -79,9 +81,23 @@ public AcquireTokenInteractiveHandler(RequestData requestData, Uri redirectUri,
             this.UniqueId = userId.UniqueId;
             this.DisplayableId = userId.DisplayableId;
             this.UserIdentifierType = userId.Type;
-            this.LoadFromCache = (requestData.TokenCache != null && parameters != null && PlatformPlugin.PlatformInformation.GetCacheLoadPolicy(parameters));
             this.SupportADFS = true;
 
+            claims = ProcessClaims(extraQueryParameters, claims);
+            if (!String.IsNullOrEmpty(claims))
+            {
+                this.LoadFromCache = false;
+
+                PlatformPlugin.Logger.Verbose(CallState, string.Format(CultureInfo.InvariantCulture,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this formatting call and simply use the input string. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS3457)